rpc/grpc/plugin_impl_grpc.go

```diff
@@ -86,31 +89,48 @@ func (p *Plugin) Init() (err error) {
 			p.metrics = grpc_prometheus.DefaultServerMetrics
 		}
 
-		// get server options from config
-		opts := p.Config.getGrpcOptions()
+		var unaryChain []grpc.UnaryServerInterceptor
+		var streamChain []grpc.StreamServerInterceptor
 
-		// add custom server options
-		opts = append(opts, p.serverOpts...)
+		// Rate limiting middleware
+		if p.limiter == nil {
```

Comment:
Does this mean that the limiter will be enabled by default, with rate and burst size hardcoded in `defaultRateLimiter()`? I'm not sure this is a good idea, people unaware of the limiter may get confused. I would rather have it disabled by default, and configurable (enabled/disabled, rate & burst size) via config file.

Reply:
I guess you are right, will fix.